src/core/lib/iomgr/ev_posix.h

```diff
@@ -33,10 +35,8 @@
 #include "src/core/lib/iomgr/pollset_set.h"
 #include "src/core/lib/iomgr/wakeup_fd_posix.h"
 
-#define GRPC_FD_TRACE(format, ...)                                      \
-  if (GRPC_TRACE_FLAG_ENABLED(fd_trace)) {                              \
-    LOG(INFO) << "(fd-trace) " << absl::StrFormat(format, __VA_ARGS__); \
-  }
+#define GRPC_FD_TRACE() \
+  LOG_IF(INFO, GRPC_TRACE_FLAG_ENABLED(fd_trace)) << "(fd-trace) "
```

Comment:
nit: This macro should be deleted and replaced with `GRPC_TRACE_LOG(fd, INFO) << "(fd-trace) " <<`. We can do that in another pass.